lib/mozart_fetcher/http_client.ex

```diff
@@ -39,6 +39,40 @@ defmodule HTTPClient do
     client.get(endpoint, headers, options)
   end
 
+  defp handle_response({:ok, response}) do
+    {:ok,
+     %HTTPoison.Response{
```

Comment:
how come we don't use something like [`Map.put`](https://hexdocs.pm/elixir/Map.html#put/3) to update the body instead of rebuilding the response struct?

Reply:
Yep, rebuilding a HTTPoison.Response binds us to HTTPoison, making migrating to a different web client library harder

Reply:
I did think there would be a better way. Thank you. I have added `Map.put` now.